pilot/pkg/model/telemetry.go

```diff
@@ -100,6 +109,7 @@ func getTelemetries(env *Environment) (*Telemetries, error) {
 		RootNamespace:          env.Mesh().GetRootNamespace(),
 		meshConfig:             env.Mesh(),
 		computedMetricsFilters: map[metricsKey]interface{}{},
+		computedLoggingConfig:  map[loggingKey]*LoggingConfig{},
```

Comment:
nit:

```suggestion
		computedLoggingConfig:  map[loggingKey]LoggingConfig{},
```

the LoggingConfig type is small, making it a pointer adds some complexity (can the map hold nil, can we mutate it, etc)

Reply:
fixed, can we remove the `DNM`?cc @howardjohn